### PR TITLE
PR#250 Iosxe/show cts environment data

### LINKED ISSUE #18: 7 - show cts environment-data
```
gw1>show cts environment-data 
CTS Environment Data
====================
Current state = COMPLETE
Last status = Successful
Local Device SGT:
  SGT tag = 0-15:Unknown
Server List Info:
Installed list: CTSServerList1-0085, 1 server(s):
  Server: 10.1.100.3, port 1812, A-ID 20FB30FE05AAA5FE90A340CAAAEDAAA0
          Status = ALIVE
          auto-test = FALSE, keywrap-enable = FALSE, idle-time = 60 mins, deadtime = 20 secs
Security Group Name Table:
    0-15:Unit0
    2-12:Unit1
    3-10:Unit2
    4-11:Device11
    3215-08:K2
    9999-06:Q1
    68-10:North
    5016-00:Quarantine
Environment Data Lifetime = 86400 secs 
Last update time = 12:25:11 UTC Mon Jul 20 2020
Env-data expires in   0:12:05:21 (dd:hr:mm:sec)
Env-data refreshes in 0:12:05:21 (dd:hr:mm:sec)
Cache data applied           = NONE
State Machine is running
gw1>

### PR DESCRIPTION
**Pull request by reustaquio22**
Parser for Iosxe/show cts environment data

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_cts.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
+        else:
+            out = output
+
+        cts_env_dict = {}
+
+        current_state_capture = re.compile(r"^Current\s+state\s=\s+(?P<state>.*$)", re.MULTILINE)
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_cts.py]: I don't think you need multiline.

File: src/genie/libs/parser/iosxe/show_cts.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+    schema = {
+        "cts_env": {
+          "current_state": str,
+          "last_status": str,
+          Optional("sgt_tags"): str,
+          Optional("tag_status"): str,
+          Optional("server_list_name"): str,
+          Optional("server_count"): int,
+          Optional("servers"): list
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_cts.py]: please adjust to be a key, and get server info

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 1 above)
Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_cts.py]: I meant you do not need multiline at all (for any of them)

**Comment by itdependsnetworks** [comment]
@reustaquio22 remove all multiline please. 

After that, please put in PR and link them here.

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_cts.py]: This has been updated

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_cts.py]: This has been updated.

**Comment by reustaquio22** [comment]
[Cisco PR #209](https://github.com/CiscoTestAutomation/genieparser/pull/209)

**Comment by itdependsnetworks** [comment]
Updated PR is https://github.com/CiscoTestAutomation/genieparser/pull/250

**Comment by itdependsnetworks** [comment]
Completed in: https://github.com/CiscoTestAutomation/genieparser/pull/250

**Pull request closed without merging**